### PR TITLE
form styling, validation and handling of course rankings

### PR DESCRIPTION
**Pull request by naeem91**
form styling, validation and handling of course rankings

### CODE REVIEW

**Comment by wajeeha-khalid** [comment]
@amir-qayyum-khan @naeem91  is it ok if I merge this PR into `jia/registration` branch to get consolidated code at one place?

**Comment by amir-qayyum-khan** [comment]
if you can test it, then it is ok,
because i havent tested it yet

**Comment by amir-qayyum-khan** [comment]
@naeem91
- i selected course 2 courses from `Check mark the courses you studied` it is giving me error ``Check mark the courses you studied: Select a valid choice``
- I am getting same error for `Mark the technology you'd like to work on/learn`
- It should allow me as much checkboxes i want to check.
- i think current issue is that we it made all check box required
 
screen shot 2017-02-06 at 11 16 47 am
screen shot 2017-02-06 at 11 17 08 am

**Comment by amir-qayyum-khan** [comment]
@naeem91 please add disclaimer to warn students that they can not give exam without filling this form

**Comment by amir-qayyum-khan** [comment]
Please make the question statement bit prominent, may be you can increase font weight?

screen shot 2017-02-06 at 11 22 27 am

File: common/djangoapps/student/arbisoft/constants.py [hunk 1]
```diff
@@ -58,4 +58,12 @@
...
     (OTHER, OTHER),
+)
+
+EXPERTISE_RANKING = (
+    (1, 1),
+    (2, 2),
+    (3, 3),
+    (4, 4),
+    (5, 5)
 )
```
Comment (amir-qayyum-khan) [inline common/djangoapps/student/arbisoft/constants.py]: @naeem91 please add new line here

File: lms/templates/arbisoft_survey.html [hunk 2]
```diff
@@ -36,43 +67,102 @@
...
+                <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
+                %for element in profile:
+                    %if element.name == 'other_studied_course':
+                        <div class="form-field">
+                            %for course in courses:
+                                <label>${course.label}</label>
+                                %for field in course:
+                                    <div class="form-field">
+                                        <label for="${field.id_for_label}">
+                                            ${field.tag()}
```
Comment (amir-qayyum-khan) [inline lms/templates/arbisoft_survey.html:117]: what are you doing for other checkbox, 
are you saving others data in [these fields](https://github.com/amir-qayyum-khan/edx-platform/pull/16/files#diff-55b798ee23a7fde8d1103408afcd0f16)? Please explain.

**Review by amir-qayyum-khan — CHANGES_REQUESTED** [review]

**Comment by amir-qayyum-khan** [comment]
Radio buttons are so small, can you increase its size
screen shot 2017-02-06 at 11 44 20 am

**Comment by wajeeha-khalid** [comment]
Why are we using forms for courses and technolgies? should not they be multi-select fields?
It makes sense to use form/formset for ranks as it is two dimensional but why for courses and technologies?

**Pull request merged**